### PR TITLE
[eclipse/xtext#1520] update to gradle 5.6.1

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1520] update to gradle 5.6.1
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.full/org.xtext.example.full.parent/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.1-bin.zip
```
Comment (szarnekow) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.full/org.xtext.example.full.parent/gradle/wrapper/gradle-wrapper.properties]: For the examples, we don't have a gradlew file in the repo that would need to be adjusted, too?

Comment (cdietrich) [reply org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.full/org.xtext.example.full.parent/gradle/wrapper/gradle-wrapper.properties]: will check

Comment (cdietrich) [reply org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.full/org.xtext.example.full.parent/gradle/wrapper/gradle-wrapper.properties]: good catch, will update the pr

**Comment by cdietrich** [comment]
@szarnekow updated the PR

**Pull request merged**